app/models/agents/twitter_stream_agent.rb

```diff
@@ -61,6 +63,15 @@ def validate_options
              options['generate'].present?
         errors.add(:base, "expected_update_period_in_days, generate, and filters are required fields")
       end
+
+      unless options[:include_retweets].present?
```

Comment:
If you want to make the field non-optional we need a database migration that changes all existing Agents to `include_retweets: false`. Your logic changes already work as expected when `include_retweets` is unset, so you could remove this check. https://travis-ci.org/huginn/huginn/jobs/329293442

Reply:
Ahh, cool.  I had no idea! 

I can remove that check simple enough.  But a follow up question - When you say "your logic changes already work as expected when ```include_retweets``` is unset" - wouldn't ruby throw some sort of keyError (I come with more python experience than anything) when it goes to do the check if it's set to false?

Reply:
In ruby the default return value of unset hash keys is `nil` which is "falsy" in conditions, because you are explicitly checking if the value is set to `false` it works.